prow/cron/cron.go

```diff
@@ -147,7 +147,7 @@ func (c *Cron) addPeriodic(p config.Periodic) error {
 
 // addJob adds a cron entry for a job to cronAgent
 func (c *Cron) addJob(name, cron string) error {
-	id, err := c.cronAgent.AddFunc(cron, func() {
+	id, err := c.cronAgent.AddFunc("TZ=UTC "+cron, func() {
```

Comment:
Can you try this with our scalability release jobs you recently moved to prow? To verify if it's WAI.

Reply:
https://k8s-testgrid.appspot.com/sig-testing-canaries#kubemark-canary-prow the canary job is WAI

Reply:
Awesome!

Reply:
sent https://github.com/kubernetes/test-infra/pull/5600 for rest of the scalability jobs :-)